pkgdown/_pkgdown.yml

```diff
@@ -1,4 +1,4 @@
-url: https://globalgov.github.io/qCreate/
+url: https://globalgov.github.io/manypkgs/
```

Comment:
Can you just please run pkgdown::build_site() locally to make sure all is good to go with the website? Thank you.

Reply:
Done!